### PR TITLE
Modified the ORE save endpoint so that it saves the files to a different collection.

### PR DESCRIPTION
**Pull request by slr71**

### CODE REVIEW

File: src/data_info/services/metadata.clj [hunk 1]
```diff
@@ -323,19 +323,29 @@
   (when-not (exists? cm path)
     (.close (output-stream cm path))))
 
+(defn- d1-metadata-dir-path
+  "Builds a path to the dataone metadata directory for a data set."
+  [data-set-path]
+  (ft/path-join
+   (ft/dirname (ft/dirname data-set-path))
+   (cfg/d1-metadata-dirname)
+   (ft/basename data-set-path)))
```
Comment (slr71) [inline src/data_info/services/metadata.clj]: The path to the data sets in the commons repository is `/iplant/home/shared/commons_repo/curated/<dataset>` and the path to the metadata directories is going to be `/iplant/home/shared/commons_repo/curated_metadata/<dataset>`. Getting the folder path this way is a little clunky, but I wanted to be able to test this endpoint using directories outside of the Data Commons repository.

Comment (psarando) [reply src/data_info/services/metadata.clj]: I think the code could benefit from some of this info in the function docs.

At first glance, I wondered what might happen if `data-set-path` was nested under another folder in `/iplant/home/shared/commons_repo/curated`, but your comment here says that is not to be expected.

So something brief in the function docs stating that `data-set-path` should always be in the top-level of the `curated` data commons directory, allowing `d1-metadata-dir-path1` to always be located in top-level of the data commons `d1-metadata-dirname` directory. Or something like that. It's Friday and updating these docs could be saved for another PR next week 😆

Comment (slr71) [reply src/data_info/services/metadata.clj]: That sounds reasonable to me. I'll update the doc comments now.

**Comment by slr71** [comment]
Thanks for the reviews!

**Pull request merged**